### PR TITLE
Adding homepage

### LINKED ISSUE #373: Add home page to function runtime
Today, hitting the root responds with a 204 (no content), which is pretty nasty when done through a browser. Let's add some kind of root page like we have for mobile sites and blank web apps that tells the user what it's all about.

### PR DESCRIPTION
**Pull request by fabiocav**
Resolves #373 

This is a preview of the default content:
https://htmlpreview.github.io/?https://github.com/fabiocav/azure-webjobs-sdk-script/blob/homepage/src/WebJobs.Script.WebHost/Home.html

### CODE REVIEW

File: src/WebJobs.Script/EnvironmentSettingNames.cs [hunk 1]
```diff
@@ -14,5 +14,6 @@ public static class EnvironmentSettingNames
         public const string AzureWebsitePlaceholderMode = "WEBSITE_PLACEHOLDER_MODE";
         public const string AzureWebsiteHomePath = "HOME";
         public const string AzureWebJobsScriptRoot = "AzureWebJobsScriptRoot";
+        public const string AzureWebJobsDisableHomepage = "AzureWebJobsDisableHomepage";
```
Comment (fabiocav) [inline src/WebJobs.Script/EnvironmentSettingNames.cs]: This app setting gives users the ability to disable the homepage if desired.

**Comment by fabiocav** [comment]
@lindydonna is working on verbiage approval. Won't merge until we have the ok.

**Comment by mathewc** [comment]
I made some comments on verbiage in the issue, e.g. replacing "experience" with "platform".

**Comment by fabiocav** [comment]
@mathewc I agree with you on the verbiage change. I'll follow up with @lindydonna as the verbiage might end up significantly different based on what we get back.

**Comment by lindydonna** [comment]
@fabiocav Here's the wording you should use:

> Azure Functions is an event-based serverless compute experience to accelerate your development.

Please also add the following fwlink:

[Learn more](https://go.microsoft.com/fwlink/?linkid=832724)

**Comment by fabiocav** [comment]
@lindydonna can you sign off on the last page update so I can merge?

@mathewc looks like "experience" has to stay :(

**Comment by lindydonna** [comment]
@fabiocav Looks fantastic 🚢 

@mathewc The term "experience" was intentional. Talk to ChrisA and Yochay.

**Comment by mikezks** [comment]
@fabiocav I changed FUNCTIONS_EXTENSION_VERSION to ~0.8 in Azure Portal, but the homepage does not show up for domain root requests.

**Comment by fabiocav** [comment]
@mikezks 0.8 hasn't been rolled out to production quite yet. We're performing final tests and should have that up soon.

**Comment by fabiocav** [comment]
Well, looks like the deployment is done! (things move quickly 😄)

Give it a try again and you should be able to see that (you may need to restart your site if it was running with the 0.8 setting the the deployment hit)

**Comment by mikezks** [comment]
Oh, I see, of course - I wasn't sure if a new version on github gets automatically deployed to Azure Portal.
Therefore I openend the following feature request: https://github.com/Azure/azure-webjobs-sdk-script/issues/842

Yes, I tested it and it works now. Great progress, guys! :simple_smile:

**Pull request merged**